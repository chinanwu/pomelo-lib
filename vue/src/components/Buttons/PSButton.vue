<template>
  <PButton c-class="PSButton">
    <template #content>
      <slot name="content"></slot>
    </template>
  </PButton>
</template>

<script>
import PButton from "@/components/Buttons/PButton";

export default {
  name: "PSButton",
  components: {
    PButton
  }
};
</script>
