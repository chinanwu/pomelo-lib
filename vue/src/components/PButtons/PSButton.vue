<template>
  <PButton class="PSButton">
    <template #content>
      <slot name="content"></slot>
    </template>
  </PButton>
</template>

<script>
import PButton from "@/components/PButtons/PButton";

export default {
  name: "PSButton",
  components: {
    PButton
  }
};
</script>
