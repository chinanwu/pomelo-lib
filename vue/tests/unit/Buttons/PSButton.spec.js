import { shallowMount } from "@vue/test-utils";
import PSButton from "@/components/Buttons/PSButton";

describe("PSButton", () => {
  describe("rendering", () => {
    it("renders PSButton", () => {
      const wrapper = shallowMount(PSButton, {
        slots: {
          content: "Test"
        }
      });
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
