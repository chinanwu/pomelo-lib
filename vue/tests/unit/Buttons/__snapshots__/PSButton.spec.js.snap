// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PSButton rendering renders PSButton 1`] = `<p-button-stub />`;
